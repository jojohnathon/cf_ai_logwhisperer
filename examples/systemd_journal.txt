Jun 21 12:11:03 laptop NetworkManager[1134]: <warn>  [1655818263.2168] device (wlan0): Activation: failed for connection 'Campus WiFi'
Jun 21 12:11:05 laptop wpa_supplicant[1073]: wlan0: CTRL-EVENT-DISCONNECTED bssid=de:ad:be:ef:00:01 reason=4
Jun 21 12:11:10 laptop kernel: iwlwifi 0000:00:14.3: Microcode SW error detected. Restarting 0x82000000.
