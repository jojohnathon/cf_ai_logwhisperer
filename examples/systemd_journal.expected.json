{
  "summary": "Wi-Fi connection to Campus WiFi is failing due to driver microcode errors causing disconnects.",
  "anomalies": [
    "NetworkManager activation failure on wlan0",
    "wpa_supplicant disconnect reason=4",
    "iwlwifi microcode software error"
  ],
  "assumptions": [
    "Driver reload or firmware update will help"
  ]
}
